customers-service/src/main/java/com/petclinic/customers/businesslayer/PetServiceImpl.java

```diff
@@ -66,6 +66,7 @@ public Pet CreatePet(PetRequest petRequest, int ownerId)
             pet.setName(petRequest.getName());
             pet.setBirthDate(petRequest.getBirthDate());
             pet.setType(petRequest.getType());
+            pet.setImageId(1);
```

Comment:
To verify, pets also have a default image set?

Reply:
yes